scaladoc/src/dotty/tools/scaladoc/tasty/SymOps.scala

```diff
@@ -100,7 +100,7 @@ object SymOps:
         Flags.Case -> Modifier.Case,
       ).collect {
         case (flag, mod) if sym.flags.is(flag) => mod
-      } ++ Seq(Modifier.Implicit).filter(_ => sym.isImplicitClass)
```

Comment:
Is `isImplicitClass` used anywhere else?

Reply:
It's used in InkuireSupport :)

Reply:
Ok, can that be rewritten as well now?